com.brave.Brave.yaml

```diff
@@ -0,0 +1,80 @@
+app-id: com.brave.Brave
+base: org.electronjs.Electron2.BaseApp
+base-version: '19.08'
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+command: chrome
+separate-locales: false
+build-options:
+  no-debuginfo: true
+finish-args:
+  - '--device=all'
+  - '--share=ipc'
+  - '--share=network'
+  - '--socket=pulseaudio'
+  - '--socket=x11'
+  - '--talk-name=org.freedesktop.FileManager1'
+  - '--talk-name=org.freedesktop.Notifications'
+  - '--talk-name=org.freedesktop.secrets'
+  - '--talk-name=org.freedesktop.UPower'
+  - '--talk-name=org.gnome.SessionManager'
+  - '--own-name=org.mpris.MediaPlayer2.*'
+  - '--filesystem=host-etc'
+  - '--filesystem=xdg-documents'
+  - '--filesystem=xdg-download'
+  - '--filesystem=xdg-music'
+  - '--filesystem=xdg-videos'
+modules:
+  - shared-modules/libsecret/libsecret.json
+
+  - name: chrome
+    buildsystem: simple
+    build-commands:
+      - 'install -Dm 755 chrome.sh /app/bin/chrome'
+      - 'install -Dm 755 apply_extra.sh /app/bin/apply_extra'
+      - 'install -Dm 755 stub_sandbox.sh /app/bin/stub_sandbox'
+      - 'install -Dm 644 -t /app/share/applications com.brave.Brave.desktop'
+      - 'install -Dm 644 -t /app/share/metainfo com.brave.Brave.metainfo.xml'
+      - |
+          for icon in 16 24 32 48 64 128 256; do
+            install -Dm 644 com.brave.Brave-$icon.png /app/share/icons/hicolor/${icon}x${icon}/apps/com.brave.Brave.png
+          done
+
+      # For extra-data to be able to extract the .deb
+      - 'install -Dm 755 /{usr,app}/bin/ar'
+      - 'cp /usr/lib/$(gcc --print-multiarch)/libbfd-*.so /app/lib'
+    sources:
+      - type: extra-data
+        # From https://brave-browser-apt-release.s3.brave.com/dists/stable/main/binary-amd64/Packages
+        url: https://brave-browser-apt-release.s3.brave.com/pool/main/b/brave-browser/brave-browser_1.14.84_amd64.deb
+        sha256: 3338e0723cf1186a8bd6a7cbfe5045773eedfa00f798c2b4aecd88850842832e
+        size: 75314612
+        filename: chrome.deb
```

Comment:
I know it doesn't matter, but I would rename anything-chrome into `brave` — across all files.

Reply:
Done, tell me if something weird happened.